core-contracts/contracts/process_template.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -euo pipefail
+
+if (( $# < 3 )); then
+    echo "Usage: $0 <input-directory> <output-directory> <yaml-files...>"
+    exit 1
+fi
+
+in_dir="$1"
+shift 1
+
+out_dir="$1"
+shift 1
+
+# The rest of the args will be treated as YAML files
+yaml_files=("$@")
+shift $#
+
+# Create tmpfile for YAML, and remove it when script exits
+yaml_tmp=$(mktemp --suffix .yaml)
```

Comment:
For compatibility with BSD `mktemp` this can be:
```suggestion
yaml_tmp=$(mktemp).yaml
```

Reply:
This is not ideal as a tempfile without the ".yaml" suffix is still created and never deleted. How about this instead?

```suggestion
yaml_tmp=$(mktemp /tmp/mustache-template.XXXXXXXX.yaml)
```